build-aux/cmd/list.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/output"
+
+	"github.com/YourFin/binappend"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(listCmd)
+}
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List data files embedded in the crc binary",
+	Long:  `List all the data files wich were embedded in the crc binary`,
+	Run: func(cmd *cobra.Command, args []string) {
+		runList(args)
+	},
+}
+
+func runList(args []string) {
+	if len(args) != 1 {
+		logging.Fatalf("list takes exactly one argument")
+	}
+	binaryPath := args[0]
+	output.Outln("List")
```

Comment:
This is not required otherwise it will show as below.
```
List
Data files embedded in out/macos-amd64/crc:
	 crc-driver-hyperkit
	 crc_hyperkit_4.2.12.crcbundle
	 hyperkit
	 oc.tar.gz
```